lib/global_stats.rb

```diff
@@ -1,13 +1,95 @@
 class GlobalStats
   include StatsHelpers
 
+  GOVUK_ROOT_CONTENT_ID = "f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a".freeze
+
+  BLACKLIST_DOCUMENT_TYPES = %w[
```

Comment:
I think it would be interesting to look at using supertypes for this instead of hardcoding the `document_type`. That will prevent this list from getting out  of date.

The new "User need" supertype might be a candidate: 

https://docs.publishing.service.gov.uk/document-types.html#user-need

Reply:
Yes I believe that it's on our horizon, and it is something that Sona and Graeme are working on (https://trello.com/c/msRhQ050/115-identify-in-scope-untagged-content-to-test-level-2-concept-coverage). I've had a look at the "User need" supertype but it doesn't seem to be well matched for our use case. At least there seems to be a fair few items that show on both the "User need" (whitelist) and this blacklist.

I don't think this should hold us up getting these metrics out, but it's something we should revisit when the Trello card linked is finished.